docs/user-guide/queries.md

```diff
@@ -115,126 +122,134 @@ Other strings may be defined for specific federates.
++--------------------------+-------------------------------------------------------------------------------------+
+| ``data_flow_graph``      | a representation of the data connections from all interfaces in a federation [JSON] |
++--------------------------+-------------------------------------------------------------------------------------+
+| ``queries``              | list of dependent objects [sv]                                                      |
++--------------------------+-------------------------------------------------------------------------------------+
+|``version_all``           | data structure with the version strings of all broker components [JSON]             |
++--------------------------+-------------------------------------------------------------------------------------+
+| ``version``              | the version string for the helics library [string]                                  |
++--------------------------+-------------------------------------------------------------------------------------+
+| ``counter``              | A single number with a code, changes indicate federation changes [string]           |
++--------------------------+-------------------------------------------------------------------------------------+
+| ``global_time_debugging``| return detailed time debugging state [JSON]                                         |
++--------------------------+-------------------------------------------------------------------------------------+
+| ``global_flush``         | a query that just flushes the current system and returns the id's [JSON]            |
++--------------------------+-------------------------------------------------------------------------------------+
 ```
 
-`federate_map`, `dependency_graph`, `global_time`,`global_state`, and `data_flow_graph` when called with the root broker as a target will generate a JSON string containing the entire structure of the federation. This can take some time to assemble since all members must be queried.
+`federate_map`, `dependency_graph`, `global_time`,`global_state`,`global_time_debugging`, and `data_flow_graph` when called with the root broker as a target will generate a JSON string containing the entire structure of the federation. This can take some time to assemble since all members must be queried. `global_flush` will also force the entire structure along the ordered path which can be quite a bit slower.
 
 ## Usage Notes
 
-Queries that must traverse the network travel along priority paths. The calls are blocking, but they do not wait for time advancement from any federate and take priority over regular communication.
+Queries that must traverse the network travel along priority paths unless specified are otherwise with a sequencing mode. The calls are blocking, but they do not wait for time advancement from any federate and take priority over regular communication.
```

Comment:
```suggestion
Queries that must traverse the network travel along priority paths unless specified otherwise with a sequencing mode. The calls are blocking, but they do not wait for time advancement from any federate and take priority over regular communication.
```